webroot/src/components/Users/UserRow/UserRow.vue

```diff
@@ -146,7 +176,16 @@
             <div class="cell affiliation"></div>
             <div class="cell states"></div>
             <div class="cell account-status"></div>
+            <div class="cell row-actions"></div>
         </div>
+        <transition mode="out-in">
+            <div v-if="isModalDisplayed">
```

Comment:
We seem to implement modals differently in our two features, Im curious on the pros / cons of each

Reply:
This user-edit modal is less like a confirmation modal and more like a user-detail page. I didn't really need much of anything from our confirmation-style modal, I just needed a custom row UI that would toggle. For instance, there was initially talk of just making this part of the row expand / collapse.